src/api/auth.ts

```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+import {API_URL, API_VERSION} from "@/constants";
+
+export async function getToken() {
+	const token = sessionStorage.getItem('token')
```

Comment:
It is quite interesting solution. Could you explain please, why do you use sessionStorage instead of cookie?

Reply:
1. I am more comfortable working with storage than with cookies
2. This way, the token is only known to the tab where it was received.